### DISCUSSION TITLE
How to set ->recordClasses for boolean key?

### DISCUSSION THREAD

**Discussion by cleargoal** [filamentphp/filament#12499] [Help]
### Package

Table builder

### Package Version

v3.2

### How can we help you?

I would like to colorize a table row using this example from the page https://filamentphp.com/docs/3.x/tables/advanced
```
 return $table
        ->recordClasses(fn (Model $record) => match ($record->status) {
            'draft' => 'opacity-30',
            'reviewing' => 'border-s-2 border-orange-600 dark:border-orange-300',
            'published' => 'border-s-2 border-green-600 dark:border-green-300',
            default => null,
        });
```
but I need a boolean key like this
`->recordClasses(fn (Model $record) => match ($record->amount < 0) {`
or like this
`$record->amount < 0 => 'bg-red-50',

Explain to me please, is it possible and how actually to realize it?

My current code doesn't work:
```
return $table
            ->recordClasses(fn (Model $record) => match ($record->amount < 0) {
                $record->amount < 0 => 'bg-red-50',
                default => null,
            })
```

**Comment by leandrocfe** [comment]
```php
->recordClasses(fn (Model $record) => match (true) {
    $record->amount <0 => 'bg-red-500',
    default => ''
})
```

_These classes are not automatically compiled by Tailwind CSS. If you want to apply Tailwind CSS classes that are not already used in Blade files, you should update your content configuration in tailwind.config.js to also scan for classes inside your directory: './app/Filament/**/*.php'_

You should also create a custom theme: https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme

**Reply by cleargoal** [reply]
**Thank you very much!**
